### PR TITLE
For Preston

### PR DESCRIPTION
**Pull request by soondubusoup**

### CODE REVIEW

File: db/init_db.js [hunk 1]
```diff
@@ -27,11 +29,14 @@ async function buildTables() {
 async function populateInitialData() {
   try {
     const product1 = await createProduct({inventory: 5, brand: 'Nike', colorway: 'blue', shoeName: 'Air', retailPrice: 50, inStock: true, releaseDate: '2004-03-04', thumbnail: "image", resellPrice: 30});
+    console.log("we made it")
```
Comment (MidnightCoder06) [inline db/init_db.js:32]: let's remove these console.logs

File: db/orders.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    }catch(error){
+        throw error
+    }
+}
+
+
+
+
+
+
```
Comment (MidnightCoder06) [inline db/orders.js:91]: remove this empty space

File: sneakerapi/SneakerData.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+const brands = ["jordan", "nike", "adidas", "crocs", "gucci", "louisvuitton"]
+
+// brands.forEach( async (brand, idx) => {
+//     console.log("----", brand, "----")
+//     await sneaks.getProducts(brand, 50, function(err, products){
+
+
+//         products.forEach((product, idx) => {
+
```
Comment (MidnightCoder06) [inline sneakerapi/SneakerData.js:13]: remove these comments

File: src/components/App.js [hunk 4]
```diff
@@ -23,11 +54,22 @@ const App = () => {
   });
 
 return (
-  <BrowserRouter>
+  <div>
+    <Route exact path='/products'>
+      <Products products={products} token={token} />
+    </Route>
+    {/* <Test /> */}
```
Comment (MidnightCoder06) [inline src/components/App.js:61]: remove?

File: src/components/Test.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+                  body: {
+                      firstName: "Xavier",
+                      lastName: "Florex",
+                      email: "Xavier514@gmail.com",
+                      username: "Xavier514",
+                      password: "xavierXavier"
+                  } });
+                  console.log(user, "it worked");
+                //   if (user) {
+                //     // localStorage.setItem("token", token);
```
Comment (MidnightCoder06) [inline src/components/Test.js:22]: remove?

**Review by rubalkaur1903 — COMMENTED** [review]
Working on it

**Review by MidnightCoder06 — APPROVED** [review]

**Pull request merged**